lib/src/actions.dart

```diff
@@ -23,9 +23,11 @@ class CustomSlidableAction extends StatelessWidget {
     this.backgroundColor = _kBackgroundColor,
     this.foregroundColor,
     this.autoClose = _kAutoClose,
+    this.shape = const RoundedRectangleBorder(),
+    this.side = BorderSide.none,
```

Comment:
It would be better if we let them null by default so that we can easily extend this class and keep the default values without redefining them.

Reply:
thank you, fixed